Images/index.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+    <title>Athaliah Portflio SetUp</title>
+ <link rel="stylesheet" href="style.css">
```

Comment:
Kindly fix the path to this css file. Currently they do not show the styles you implemented. 
This is your project currently.
<img width="1002" alt="Screenshot 2022-10-28 at 17 02 10" src="https://user-images.githubusercontent.com/48546992/198633945-40b0b424-b165-490d-986f-5702b19d4310.png">


This is the figma template
<img width="369" alt="Screenshot 2022-10-28 at 11 23 07" src="https://user-images.githubusercontent.com/48546992/198633084-e4f18421-6c1a-4d9d-8504-296c3dbe9ff6.png">

I suggest you either move the index.html to the root folder or fix the path.